client/src/components/ListsDisplay.js

```diff
@@ -52,22 +50,23 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const demoListsArray = [
-  { listTitle: "Clothes", itemCount: 34, img: clothes },
-  { listTitle: "Furniture", itemCount: 12, img: furniture },
-  { listTitle: "Luxury", itemCount: 8, img: luxury },
-  { listTitle: "Clothes", itemCount: 34, img: clothes },
-  { listTitle: "Furniture", itemCount: 12, img: furniture },
-  { listTitle: "Luxury", itemCount: 8, img: luxury },
-  { listTitle: "Clothes", itemCount: 34, img: clothes },
-  { listTitle: "Furniture", itemCount: 12, img: furniture },
-  { listTitle: "Luxury", itemCount: 8, img: luxury },
-];
-
 const ListsDisplay = (props) => {
   const scrollRef = useHorizontalScroll();
   const { profile, addListOpen, changeAddListOpen } = props;
   const classes = useStyles();
+  const userId = useContext(UserContext).userId;
+  const [lists, setLists] = useState([]);
+
+  const getLists = async () =>{
+    const res = await fetch(`/lists?user_id=${userId}`)
+    const json = await res.json()
+    setLists(json);
+  }
+
+    useEffect(() => {
+    getLists();
+  }, [userId]);
+
```

Comment:
So this one is being done in two places, so let's move it to either the parent component or to the list context